src/sandstorm/hack-session.capnp

```diff
@@ -65,45 +65,9 @@ interface HackSessionContext @0xe14c1f5321159b8f
 
   generateApiToken @3 (petname :Text, userInfo :Grain.UserInfo, expires :UInt64 = 0)
       -> (token :Text, endpointUrl :Text, tokenId :Text);
-  # Generates a new API token which can be used to access an HTTP API exported by this application.
```

Comment:
Should this deleted comment be replaced with a comment saying this method is no longer supported?

Maybe we should rename the method to obsoleteGetApiToken?